crt_portal/cts_forms/forms.py

```diff
@@ -58,8 +58,12 @@ class Details(ModelForm):
     def __init__(self, *args, **kwargs):
         super(ModelForm, self).__init__(*args, **kwargs)
         self.fields['violation_summary'].widget.attrs['class'] = 'usa-textarea'
-        self.fields['violation_summary'].label = 'Please describe what happened in your own words'
-        self.fields['violation_summary'].help_text = ['This is your opportunity to share any additional details you think would be helpful for us to know like time, names, witnesses to the event, or anything you haven’t already stated. Document upload is not available at this time, but please describe if you have evidence you’d like to include later.', 500, 'word(s) remaining']
+        self.fields['violation_summary'].help_text = (
+            "This is your opportunity to share any additional details you think would be helpful "
```

Comment:
I am not sure why is this text broken up? This might not be great for translation but I don't have strong feelings.

Reply:
Without breaking up the text this line was 380 characters/columns long, that makes it harder to spot typos and understand diffs to the text.

Reply:
I feel auto text wrapping takes care of that. I don't think that is how we will want to break up the translation in the future so I am inclined to keep it together.

Reply:
OK! I liked the multiline string because I think it's good code style, but I'll defer to you and revert it to a long string. 👍

Reply:
Done in 15b9206066fa81e9cdc58b3b1a2b35ae700a7c93.